apps/windows/windows/NuGet.Config

```diff
@@ -6,5 +6,6 @@
   <packageSources>
     <add key="Nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="react-native" value="https://pkgs.dev.azure.com/ms/_packaging/react-native/nuget/v3/index.json" />
+    <add key="react-native-public" value="https://pkgs.dev.azure.com/ms/react-native/_packaging/react-native-public/nuget/v3/index.json" />
```

Comment:
Just curious, what does this feed have that the feed above it doesn't?

Reply:
I'm not sure exactly. I was having trouble connecting to our private NuGet feed on ADO pipeline. I think this feed publishes all versions of react-native and doesn't need authentication. 

From Danny Van Velzen: I'll quickly create a new feed react-native-public and publish all versions of react-native in there. That one should not need any authentication.